src/components/Dropdowns/AccessibleDropdown.tsx

```diff
@@ -91,18 +91,19 @@ const MainButton = styled(Button)<{ $open: boolean; $type: AccessibleDropdownPro
   ${(props) => {
     if (props.$type === "outlined") {
       return css`
-        &.ant-btn:not(:disabled) {
+        // Extra classnames to increase selector specificity; otherwise gets overridden by root styling
+        &.ant-btn.ant-btn-default:not(:disabled) {
```

Comment:
why not just add your own class? instead of multiple ant classes

Reply:
Thank you for the suggestion! Refactored this :)